src/libs/API.js

```diff
@@ -131,8 +131,9 @@ function handleExpiredAuthToken(originalCommand, originalParameters, originalTyp
         })
         .catch(() => (
 
-            // If the request did not succeed and we were not logged out requeue the original request. We are using
-            // request() here so that it we can reauthenticate again when it fails
+            // If the request did not succeed because of a networking issue or the server did not respond requeue the
+            // original request. We are using request() here so that it we can reauthenticate again if another
+            // request() has not refreshed the authToken. Network.post() does not trigger handleExpiredAuthToken().
             // eslint-disable-next-line no-use-before-define
             request(originalCommand, originalParameters, originalType)
```

Comment:
I'm still a little confused on `Network.post()` vs. `request()`

> Network.post() does not trigger handleExpiredAuthToken().

I think this needs a little more context to explain why this matters or not, because that's the piece I think I'm missing. 

Is there any kind of scenario where we can use either `Network.post()` or `request()` in both places? That would help to alleviate my confusion if we could just consistently use one or the other.